src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -2093,6 +2093,26 @@ public FixableDiagnostic ProviderDeclarationViaImportKeywordIsDeprecated(Provide
                     DiagnosticStyling.Default,
                     codeFix);
             }
+
+            public ErrorDiagnostic MalformedProviderPackage(string ociManifestPath) => new(
```

Comment:
Was this an intentional addition?

Reply:
No, this must have been removed from `main` and then snuck back in during a merge conflict resolution. Will prune in next commit.